packages/core/src/command/patient-import/patient-import-shared.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  normalizeDate,
+  normalizeEmailStrict,
+  normalizeExternalId,
+  normalizeGender,
+  normalizePhoneNumberStrict,
+  normalizeState,
+  normalizeZipCodeNew,
+  PatientImportPatient,
+  toTitleCase,
+} from "@metriport/shared";
+import { PatientPayload } from "./patient-import";
+
+const globalPrefix = "patient-import";
+
+export type FileStages = "raw" | "valid" | "invalid";
+
+function createCxJobPrefix(cxId: string, jobId: string): string {
+  return `cxid=${cxId}/jobid=${jobId}`;
```

Comment:
are we not storing date on the folder structure too?

`cxid/YYYY/MM/DD/jobid/...` or maybe just `cxid/YYYY/YYYY-MM-DD/jobid/...`

Reply:
went with `cxid={cxId}/date={YYYY-MM-DD}/jobid={jobId}`. Done.